Kalkulator_Github/Form1.cs

```diff
@@ -21,7 +21,7 @@ private void button1_Click(object sender, EventArgs e)
         {
             try
             {
-                sum.Text = (int.Parse(NumberA.Text.ToString()) + int.Parse(NumberB.Text.ToString())).ToString();
+                sum.Text = (int.Parse(NumberA.Text) + int.Parse(NumberB.Text)).ToString();
```

Comment:
Bardziej czytelnie jest, jeśli wyrażenie podzielimy na mniejsze, używając zmiennych lokalnych
```suggestion
                int a = int.Parse(NumberA.Text);
                int b = int.Parse(NumberB.Text);
                sum.Text = (a + b).ToString();
```